devel/bmake/Makefile

```diff
@@ -38,6 +39,8 @@ DEPENDS+=		pdksh-[0-9]*:../../shells/pdksh
 CONFIGURE_ARGS+=	--with-defshell=${PREFIX}/bin/pdksh
 .elif ${OPSYS} == "SunOS" && exists(/usr/bin/bash)
 CONFIGURE_ARGS+=	--with-defshell=/usr/bin/bash
+.elif ${OPSYS} == "UnixWare" && exists(/usr/bin/bash)
```

Comment:
Probably worth starting a combined version of this, I imagine it'll be true for others too.